tests/Tests/400__ods_dashboard/411__ods_dashboard_settings_pvc_size.robot

```diff
@@ -102,11 +102,13 @@ Verify Multiple Unsupported Size
     END
 
 Change And Apply PVC size
-    [Documentation]    Make PVC change to configmap
-    ...    and rollout deployment config
+    [Documentation]    Make PVC change to OdhDashboardConfig CR
+    ...    and restart the Notebook controller POD
     [Arguments]     ${size}
     Check If PVC Change Is Permanent    ${size}
-    Roll Out Jupyter Deployment Config
+    Run Keyword And Ignore Error    Delete Pods Using Label Selector    redhat-ods-applications    app=notebook-controller
+    Wait Until Keyword Succeeds    15 times    1 sec   Verify Operator Pod Status    redhat-ods-applications    app=notebook-controller
+    Sleep    20
```

Comment:
did you try without restarting the pod? For examplen with KFNBC when you change the User Management settings you don't have to restart pods anymore, just wait kind of 10 seconds approx. Maybe it's the same with PVC (unless you tried)

Reply:
I have tried we need to wait. at least that what i saw in my verification

Reply:
so if we need to wait, why deleting the pods?